packages/boot/package.json

```diff
@@ -32,8 +32,10 @@
     "@loopback/repository": "^0.11.3",
     "@types/debug": "0.0.30",
     "@types/glob": "^5.0.35",
+    "@types/lodash": "^4.14.109",
     "debug": "^3.1.0",
-    "glob": "^7.1.2"
+    "glob": "^7.1.2",
+    "lodash": "^4.17.10"
```

Comment:
I don't see any changes with references to lodash. Probably left in as a mistake?

Reply:
Good catch! I was using it for determining the binding name but we since switched to the static `dataSourceName` property for that ... and as such no longer need `lodash`.